runtime/runtime_test.go

```diff
@@ -180,6 +185,16 @@ type testRuntimeInterface struct {
 	recordTrace                func(operation string, location common.Location, duration time.Duration, logs []opentracing.LogRecord)
 }
 
+func (i *testRuntimeInterface) ResourceOwnerChanged(
```

Comment:
Not a big deal: maybe move this below [line#199](https://github.com/onflow/cadence/pull/1289/files#diff-74ffa81b6938856f695ba112a721503dfc5ff2e852bc65d79d15ee4868f4bb97R199):
```
// testRuntimeInterface should implement Interface
var _ Interface = &testRuntimeInterface{}
```
 Just so that this method would be with the rest of the methods

Reply:
Cleaned up in 5ad53fe21d62fae4f2e804773485b6c089f6b598